src/asciidoc/docs/guides-general/build-custom-controls.adoc

```diff
@@ -0,0 +1,210 @@
+= Building custom controls
+:page-slug: build-custom-controls
+:page-description: How to build a custom control that captures user input.
+:includespath: ../_includes
+
+The Koji platform provides controls that you can use to capture user input, such as images, text, and sounds.
+In addition, you can create your own custom controls.
+
+Examples of custom controls are tile map editors, sound enhancers, or custom avatar creators, which enhance the app's interactivity.
+
+A custom control is an application in Koji; therefore, you create one the same way you create any other Koji application.
+However, unlike a typical Koji app, a custom control has only one context (Default).
+When triggered, it appears over the parent app like a modal.
+
+Because a custom control does not include other contexts, it cannot be used as a standalone app.
+
+== Creating a custom control
+
+=== Prerequisites
+
+include::{includespath}/prereqs.adoc[tags=webdev;remixproc]
+
+=== Steps
+
+. <<_create_a_new_project,Create a new project and install the required packages.>>
+. <<_code_your_custom_control,Code your custom control.>>
+. <<_provide_a_link_to_your_custom_control,Provide a link to your custom control.>>
+. <<_test_your_custom_control,Test your custom control.>>
+
+==== Create a new project
+
+. Fork the https://withkoji.com/@eltesorokoji/a5hv/details/[Custom Control Scaffold].
```

Comment:
This week's change to navigation makes the App Store is the primary way to use apps. So, we should update our tutorials to start from an app listing rather than the developer details or the app itself. When we decide on the starting point for this page, we can update this link.

Reply:
Because the custom control scaffold isn't listed yet, I didn't have a way to point to an App Store listing.

I just got a message from Sean that he just fixed the issue. (It was a platform bug.) I'll test it, create a listing for both scaffold and tester app, then I'll update the links in the topic.

Reply:
UPDATE: I hit another issue when I tried to create listings for both the custom control and the tester app. [Details in the Jira.](https://withkoji.atlassian.net/browse/DC-617?focusedCommentId=10385)

Reply:
The scaffold and example tester app are now in the App Store and added to the ["For Developers" section](https://withkoji.com/create/for-developers) by Luis.